sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/LocalDocumentStorage.java

```diff
@@ -338,15 +306,47 @@ void updatePendingOperation(PendingOperation operation) {
         }
     }
 
-    private static ContentValues getContentValues(PendingOperation operation, long now) {
-        return getContentValues(
-                operation.getPartition(),
-                operation.getDocumentId(),
-                operation.getDocument(),
-                operation.getEtag(),
-                operation.getExpirationTime(),
-                now,
-                now,
-                operation.getOperation());
+    @NonNull
```

Comment:
Someone didn't run the formatter, the formatter can reorder methods based on static or visiblity